e2e/rbd.go

```diff
@@ -1088,6 +1088,111 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("validate PVC mounting if snapshot and parent PVC are deleted", func() {
+				// snapshot beta is only supported from v1.17+
+				if k8sVersionGreaterEquals(f.ClientSet, 1, 17) {
```

Comment:
Nit: can you add the test like this?
```golang
                                if !k8sVersionGreaterEquals(f.ClientSet, 1, 16) {
                                        Skip("pvc clone is only supported from v1.16+")
                                }
```

That way, indention is a little less, and it makes it easier to drop support for deprecated Kubernetes versions at a later time.